pcdsdevices/pseudopos.py

```diff
@@ -649,6 +624,44 @@ def fix_sync(self, confirm=True, wait=True, timeout=10):
                 for status in statuses:
                     status.wait(timeout=timeout)
 
+    def format_status_info(self, status_info):
+        """
+        Special SyncAxis handling to show all the real motors.
+        """
+        lines = []
+        if self.warn_inconsistent and not status_info['is_synced']:
+            lines.append(self.consistency_warning())
+        big_name = status_info['name']
+        lines.append(f'SyncAxis: {big_name}')
+        for attr in self._real_attrs:
+            info = status_info[attr]
+            name = get_status_value(info, 'name')
+            name = name.replace(big_name + '_', '')
+            pos = get_status_float(info, 'position', format='g')
+            units = get_status_value(info, 'units')
```

Comment:
I have not tested this very well, but I've noticed in your example you get `[N/A]` here - and I am wondering if it is because you use the `fake_lxt_ttc` and it can't find the units, or is it because you might have to get them differently like:
`"info", "setpoint", "units"`... again, i did not check very well, will test in a bit

Reply:
A closer investigation shows various issues related to this, both exactly right here as you pointed out (needs to be setpoint units) and also elsewhere where for Newport motors and Delay motors the units are not present at all in the statuses

Reply:
I've noticed that the units are not present in the statuses, they are either `None` or `''` (empty) - is that related to the `status_info()` specifically or the fact that we actually don't have units set for those devices? (i'm curious because i think i've seen this before and always thought that they are just not set for those components)

Reply:
I'm not 100% sure on this. For the `Newport` motors there is some bugginess with their metadata in general as reported by EPICS (that we are partially working around but missed a spot). For other devices like the pseudos I suspect they are victims of inconsistent handling of the units data in the `status_info()`.

As an aside, I'm tentatively planning to refactor the status info to make this less annoying to implement. We know a lot about shortcomings of the current API now than we did at the start.